config.default.toml

```diff
@@ -20,6 +20,12 @@ command = "st"
 keys = [ "Return" ]
 modifiers = [ "super" ]
 
+[autostart]
+exec = [
+	"firefox",
```

Comment:
Let's comment out these (just the "firefox" and "st" lines) so our default config doesn't try to launch programs

Reply:
oh right, I forgot about that. So, just keep empty `exec` array?

Reply:
Yeah, just put a `#` before "firefox" and "st" to comment the lines out